src/main/java/StringUtils.java

```diff
@@ -1,5 +1,13 @@
 public class StringUtils {
-    public static boolean isPositiveNumber(String str){
```

Comment:
Rule of thumb: to create a javadoc for a class and its public methods. Highlight which cases are covered, expected behavior and what kind of exceptions can be thrown.